packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx

```diff
@@ -37,7 +25,9 @@ const PaypalPaymentLegalBlurb = () => (
         ),
       }}
     >
-      <p>{getPaypalPrivacyLinkText()}</p>
+      <LinkExternal href="https://www.paypal.com/webapps/mpp/ua/privacy-full">
+        PayPal privacy policy
+      </LinkExternal>
```

Comment:
Why the markup change?  I think we probably want that p tag.

Reply:
Ouff, that was a bad assumption on my part. Thanks for picking this up. Fixed.